modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -123,6 +124,36 @@ public void testGetObjectDefinition() throws Exception {
 			objectDefinition.getRestContextPath());
 	}
 
+	@Override
+	@Test
+	public void testGetObjectDefinitionsPage() throws Exception {
+
+		// super.testGetObjectDefinitionsPage();
+
+		ObjectDefinition objectDefinition =
+			testGetObjectDefinitionsPage_addObjectDefinition(
+				randomObjectDefinition());
+
+		testGetObjectDefinitionsPage_addObjectDefinition(
+			_randomModifiableSystemObjectDefinition());
+
+		// Filter by status
+
+		Page<ObjectDefinition> objectDefinitionsPage =
+			objectDefinitionResource.getObjectDefinitionsPage(
+				null, null, "status/any(k:k eq 2)", Pagination.of(1, 2), null);
+
+		List<ObjectDefinition> objectDefinitionsPageItems =
+			(List<ObjectDefinition>)objectDefinitionsPage.getItems();
+
+		assertEquals(
+			Collections.singletonList(objectDefinition),
+			objectDefinitionsPageItems);
+
+		_assertStatusCode(
```

Comment:
This is just used once. Anyway, I think we don't need this assertion as the previous one is already comparing with the objectDefinition (included the status field)